src/Commands/SiteInstallCommands.php

```diff
@@ -33,6 +33,9 @@ public function siteInstallPostCommand($result, CommandData $commandData): Comma
       if (isset($arguments['profile'][0]) && $arguments['profile'][0] == 'acquia_cms') {
         $telemetry->setTime('rebuild_end_time');
       }
+      if (function_exists('acquia_cms_print_icon')) {
```

Comment:
@panshulK I think we should print icon only if `acquia_cms` profile is installed. So, probably this code could go inside just above function i.e. here:
```
if (isset($arguments['profile'][0]) && $arguments['profile'][0] == 'acquia_cms') {
  // print acquia_cms icon
}

Reply:
done